### PR TITLE
Added 7 icons of social media in png | #3

### LINKED ISSUE #3: [FEATURE] Social Media logos
## Description
Adding following social media logo in [`assets/logos`](https://github.com/Param302/SocialRepo/tree/main/src/assets/logos) folder.

### Requirements:
-   Icons should be of minimum `128x128` and maximum `256x256` resolution.
-   Icons must be in `png` format and transparent background.
-   Icons should be named as `social-media-name.png` (e.g. `twitter.png`, `github.png`, `instagram.png`, etc...).
-   Icon names must match the keys in [`social-links.json`](https://github.com/Param302/SocialRepo/blob/main/src/social-links.json).
-   Make sure the social media app of icon must have social (public) user profile system, and exists in [`social-links.json`](https://github.com/Param302/SocialRepo/blob/main/src/social-links.json).

## Social media Icon list
The list of social media apps/website which has *public user profile* feature is present in [`social-apps.md`](https://github.com/Param302/SocialRepo/blob/main/.github/docs/social-apps.md)

You can choose any 7 social media names which are not present in [`logos`](https://github.com/Param302/SocialRepo/tree/main/src/assets/logos) folder and listed in [`social-links.json`](https://github.com/Param302/SocialRepo/blob/main/src/social-links.json).

If there are less names listed in [`social-links.json`](https://github.com/Param302/SocialRepo/blob/main/src/social-links.json), feel free to add more names with profile URLs.

Make sure to **NOT** to choose the apps which are already done.

### How to check ❓
- Go to [`logos`](https://github.com/Param302/SocialRepo/tree/main/src/assets/logos) folder, check which are already there, if any icon which you want to work is already there, please choose another one.

_Please feel free to recommend and more apps which has social (public) user profile system._

## Style
```json
{
    "social_media_name": "link",
    "linkedin": "https://www.linkedin.com/in/<username>",
    "github": "https://www.github.com/<username>",
    "medium": "https://<username>.medium.com"
}
```

_Let's create the user life easier by adding all social media platforms logos._

### PR DESCRIPTION
**Pull request by Suvam3456**
Hello Project Admin,
I have added 7 social media icons in png format which are in  specified resolution range and updated the social-links.json and social-apps.md.

Fixes #3 

## Description
Added  social media icons in png format

---
## Issue Ticket Number
Fixes #(issue_number)

---
# Checklist:
- [ ✅] I have followed the contributing guidelines of this project as mentioned in [CONTRIBUTING](./docs/CONTRIBUTING)
- [ ✅] I have checked to ensure there aren't other open [Pull Requests](https://github.com/Param302/SocialRepo/pulls) for the same update/change?
- [✅ ] I have performed a self-review of my own code
- [ ✅] I have commented my code, particularly in hard-to-understand areas
- [✅ ] I have made corresponding changes needed to the documentation

### CODE REVIEW

File: src/assets/logos/medium.png
Comment (Param302) [inline src/assets/logos/medium.png:1]: This is not the medium icon, check [here](https://medium.design/logos-and-brand-guidelines-f1a01a733592) and upload the medium symbol.

**Review by Param302 — CHANGES_REQUESTED** [review]
Hey @Suvam3456 ,

Just change the medium logo, rest is all good to go.
After the changes, I will merge it.

File: src/assets/logos/medium.png
Comment (Suvam3456) [reply src/assets/logos/medium.png:1]: Sure

**Review by Param302 — APPROVED** [review]
Thanks for Contributing @Suvam3456 !
I am merging it to `main` branch.

Would love to see more contributions from you. :D
_Happy Contributing!_

**Pull request merged**